### PR TITLE
Outputs a partition-specific location for terrafirm artifacts

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
go codebuild go

File: buildspec.yml [hunk 1]
```diff
@@ -14,16 +14,24 @@ phases:
...
-      - echo "ARTIFACT_LOCATION=https://s3.console.aws.amazon.com/s3/buckets/${ARTIFACT_PATH}/"
+      - >  # YAML folding syntax avoids interpreting the : as a map value
+        PARTITION="$(aws sts get-caller-identity --query Arn --out text | cut -d: -f2)"
+      - |
+        if [ "$PARTITION" == "aws-us-gov" ]; then
+          ARTIFACT_LOCATION="https://${AWS_DEFAULT_REGION}.console.amazonaws-us-gov.com/s3/buckets/${ARTIFACT_PATH}/"
+        else
+          ARTIFACT_LOCATION="https://s3.console.aws.amazon.com/s3/buckets/${ARTIFACT_PATH}/"
+        fi
+      - echo "ARTIFACT_LOCATION=${ARTIFACT_LOCATION}"
```
Comment (dwc0011) [inline buildspec.yml:29]: You echo the location here and below, is that intentional?

**Review by dwc0011 — DISMISSED** [review]

**Comment by lorengordon** [comment]
go codebuild go

**Review by dwc0011 — APPROVED** [review]

**Pull request merged**